src/test/scala/nl.knaw.dans.easy.deposit/authentication/LdapAuthenticationSpec.scala

```diff
@@ -46,65 +43,75 @@ class LdapAuthenticationSpec extends TestSupportFixture with MockFactory {
   }.authentication
 
   "getUser(user,password)" should "return an active user" in {
-    ldapMocker.expectLdapAttributes(new BasicAttributes() {
-      put("dansState", "ACTIVE")
-      put("uid", "someone")
-      put("easyGroups", "abc")
+    val authentication = wire(new LdapMocker {
+      expectLdapAttributes(new BasicAttributes() {
+        put("dansState", "ACTIVE")
+        put("uid", "someone")
+        put("easyGroups", "abc")
+      })
     })
-
-    wire.authenticate("someone", "somepassword") should matchPattern {
+    authentication.authenticate("someone", "somepassword") should matchPattern {
       case Some(AuthUser("someone", Seq("abc"), UserState.active)) =>
     }
   }
 
   it should "return none for a blocked user" in {
-    ldapMocker.expectLdapAttributes(new BasicAttributes() {
-      put("dansState", "BLOCKED")
+    val authentication = wire(new LdapMocker {
+      expectLdapAttributes(new BasicAttributes() {
+        put("dansState", "BLOCKED")
+      })
     })
-
-    wire.authenticate("someone", "somepassword") shouldBe None
+    authentication.authenticate("someone", "somepassword") shouldBe None
   }
 
   it should "fail on other ldap problems" in {
-    ldapMocker.expectLdapSearch throwing new Exception("whoops")
-    ldapMocker.expectLdapClose
-
-    wire.authenticate("someone", "somepassword") should matchPattern {
+    val authentication = wire(new LdapMocker {
+      expectLdapSearch throwing new Exception("whoops")
+      expectLdapClose
+    })
+    authentication.authenticate("someone", "somepassword") should matchPattern {
       case None => // different logging than with AuthenticationException
     }
   }
 
   it should "return none for an invalid username or password" in {
-    ldapMocker.expectLdapSearch throwing new AuthenticationException()
+    val authentication = wire(new LdapMocker {
+      expectLdapSearch throwing new AuthenticationException()
+    })
+    authentication.authenticate("someone", "somepassword") shouldBe None
+  }
 
-    wire.authenticate("someone", "somepassword") shouldBe None
+  it should "fail without an ldapContext" ignore { // TODO details in comment inside
```

Comment:
@rvanheest can't find how to ignore-until-fixed

Reply:
pendingUntilFixed

Reply:
https://stackoverflow.com/a/36428115